control-plane/grpc/src/operations/volume/traits.rs

```diff
@@ -993,6 +1020,16 @@ impl From<&dyn PublishVolumeInfo> for PublishVolume {
     }
 }
 
+impl From<&dyn PublishVolumeInfo> for RepublishVolume {
+    fn from(data: &dyn PublishVolumeInfo) -> Self {
+        Self {
+            uuid: data.uuid(),
+            target_node: data.target_node(),
+            share: data.share().unwrap(),
+        }
+    }
+}
```

Comment:
Is this needed?
and also `impl PublishVolumeInfo for RepublishVolume {` ?

Reply:
Yes, this is needed since we have seperated the structures for PublishVolume and RepublishVolume.